plugins/shopify/src/plugin.ts

```diff
@@ -0,0 +1,88 @@
+import { registerCommercePlugin } from '@builder.io/commerce-plugin-tools';
+
+import Client from 'shopify-buy';
+import pkg from '../package.json';
+import appState from '@builder.io/app-context';
+
+registerCommercePlugin(
+  {
+    name: 'Shopify',
+    // should always match package.json package name
+    id: pkg.name,
+    settings: [
+      {
+        name: 'storefrontAccessToken',
+        type: 'string',
+        helperText:
+          'Required to sync, index, and cache your storefront product data to avoid rate limits',
+        required: true,
+      },
+      {
+        name: 'storeDomain',
+        type: 'text',
+        helperText: 'your-store.myshopify.com',
```

Comment:
In my experience, people sometimes get confused and only put the `your-store` part, so I'd maybe say

```suggestion
        helperText: 'Your entire store domain, such as "your-store.myshopify.com"',
```

or something like that